### PR TITLE
Fix program breaking bugs

### PR DESCRIPTION
**Pull request by zapdos26**
Fixes program breaking bugs and shortens/standardizes the length requirement

### CODE REVIEW

**Comment by zapdos26** [comment]
I'm fine with lengthening it to three and adding in the today and week into case switch.  Personally don't know what they are separated.

**Comment by zapdos26** [comment]
To be honest, it also might be wise to look into using moment for this. Much easier to understand

File: server.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const RPIA_WEB_TOKEN = process.env.RPIA_WEB_TOKEN;
 const SLACK_SIGNING_TOKEN = process.env.SLACK_SIGNING_TOKEN;
 const PORT = process.env.NODE_PORT || 3000;
 
-const whoson = t => {
+const whoson = async(t) => {
```
Comment (ddbruce) [inline server.js]: Can we make `t` a little more semantic? I realize it was like that previously

File: utilities/helpers.js [hunk 2]
```diff
@@ -19,16 +19,30 @@ const dateForRelative = (name, offset) => {
...
+      return dateForOffset(3);
+    case 'th':
+      return dateForOffset(4);
+    case 'fr':
+      return dateForOffset(5);
+    case "sa":
+      return dateForOffset(6);
+    default:
+      return null
+  }
```
Comment (ddbruce) [inline utilities/helpers.js:45]: Oooo I like this

**Review by ddbruce — APPROVED** [review]
I'll leave it up to you if you want to change `t` or not. Just a thought

**Pull request merged**